app/lib/wolf_waagen_api/import.rb

```diff
@@ -98,17 +99,47 @@ def start_date
 
     def save_series_points(series:)
       sensors = api_sensors_by_series_id(series_id: series.id)
+      interval = accumulation_interval_by_series_id(series_id: series.id)
 
       sensors.each do |sensor|
         series.points.each do |point|
+          value = accumulated_value(sensor: sensor, interval: interval, point: point)
```

Comment:
Very counter-intuitive. I would not understand that `accumulated_value` returning an accumulated value is in fact an edge case! :confused: